### ISSUE TITLE
Using multiple Piston Emitters will make an horrendous audio

### ISSUE THREAD

**Issue by underlineGalaxy** [BEEmod/BEE2-items#3065]
https://drive.google.com/file/d/1Zbc7xcj2lldoi0RnW37E37mYQ2dZA40b/view?usp=sharing

Pretty sure the game plays the audio of all the pistons at once, a solution would be to either lower down the volume (significantly) or make it so it only plays one audio when it's multiple pistons.

**Comment by JavierNovellon** [comment]
Pretty sure this is from PeTI itself? I had that problem in one of my chambers before I had BEE2; and I ended up changing the chamber because of how jarring the noise was.

If this could be fixed, it'd be great for chambers where the walls retract to show an extra puzzle. (dunno, that was what my chamber had. you finished part of the chamber and then pistons and angled panels would retract to reveal more puzzles)

**Comment by underlineGalaxy** [comment]
@JavierNovellon yes it might be an issue with the puzzlemaker itself, however there are also some annoying behaviours on the default puzzlemaker that beemod changes, so this still might come as a valid issue.

**Comment by Konclan** [comment]
This could definitely be fixed just not easily. There needs to be special compiler code that figures out what items are connected together like pistons and removes all but 1 ambient_generic. This would probably work with stuff like connecting multiple angled panels together

**Comment by TeamSpen210** [comment]
Not quite, it can be done with a custom soundscript that prohibits more than one from being started simultaneously.
